debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -335,8 +335,8 @@ protected void execute() {
                     return;
                 }
                 logger.info("Step {}: read list of available tables in each database", step++);
-                List<TableId> tableIds = new ArrayList<>();
-                //List<TableId> allTableIds = new ArrayList<>();
+                List<TableId> knownTableIds = new ArrayList<>();
+                final List<TableId> monitoredTableIds = new ArrayList<>();
```

Comment:
Can we generally use the term "captured tables" instead of "monitored tables". In particular in user-facing messages (logs, exceptions etc.), it avoids confusion to the concept of monitoring (as in keeping an eye on the system's state).